charts/dbildungs-iam-server/templates/backend-ingress.yaml

```diff
@@ -30,4 +30,41 @@ spec:
                 port:
                   number: {{ $.Values.backend.service.ports.http }}
           {{- end }}
+{{ end }}
+---
+{{if .Values.backend.ingress.healthenabled }}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ template "common.names.name" . }}-health-backend
+  namespace: {{ template "common.names.namespace" . }}
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /health
+    # authentication type
+    nginx.ingress.kubernetes.io/auth-type: basic
+    # name of the secret that contains the user/password definitions
+    nginx.ingress.kubernetes.io/auth-secret: {{ .Values.auth.secretName }}
```

Comment:
i think the comments in 45 an 47 are not adding much additional value. It is easiest to use the existing dbildungs-iam-server secret. The ingress also only seems to use the "auth" key so no other secrets would ever get exposed, still i am not sure whether creating a extra secret would the cleaner way. Has anyone an opinion on that?